compositions/cluster-k8s/big-blocks/tc-001-val-large-txs/40-vals-180kb-pfd/320mib-200ms.toml

```diff
@@ -5,19 +5,21 @@
 [global]
   plan = "celestia"
   case = "001-val-large-txs"
-  total_instances = 40
+  total_instances = 42
   builder = "docker:generic"
   runner = "cluster:k8s"
   disable_metrics = false
-
-[global.run.test_params]
-  execution-time = "10"
-  latency = "200"
-  bandwidth = "320Mib"
-  validator = "40"
-  persistent-peers = "10"
-  submit-times = "10"
-  msg-size = "180000"
+  [global.run]
+    artifact = ""
+    [global.run.test_params]
+      bandwidth = "320Mib"
+      execution-time = "10"
+      latency = "200"
+      msg-size = "180000"
```

Comment:
Blocking: Should we account for padding here? like in the other file, we used `160kb` ?

Reply:
Let's do it in another PR from this issue #143